controller/array_action/array_mediator_svc.py

```diff
@@ -761,8 +773,11 @@ def map_volume(self, volume_id, host_name):
         }
 
         try:
-            lun = self.get_first_free_lun(host_name)
-            cli_kwargs.update({'scsi': lun})
+            if connectivity_type != config.NVME_OVER_FC_CONNECTIVITY_TYPE:
+                lun = self.get_first_free_lun(host_name)
+                cli_kwargs.update({'scsi': lun})
+            else:
+                lun = config.NVME_OVER_FC_CONNECTIVITY_TYPE
```

Comment:
why is this needed?

Reply:
cause nvme is not using lun

Reply:
I still don't understand why line 780 here is needed

Reply:
so there will be an indication for it in the logs instead of just None...

Reply:
besides, I see the lun being logged later with the connectivity type, so to me the logs should be fine